assets/js/modules/pagespeed-insights/components/dashboard/DashboardPageSpeed.js

```diff
@@ -33,9 +31,9 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+import { Button, ProgressBar, Tab, TabBar } from 'googlesitekit-components';
 import API from 'googlesitekit-api';
 import Data from 'googlesitekit-data';
-import { Button, ProgressBar } from 'googlesitekit-components';
```

Comment:
Since this is such a minor change, I've pushed it myself. I hope that is okay with you, thanks!

Commit: https://github.com/google/site-kit-wp/pull/7265/commits/57e1ff8a43182f0bae95000bc8fd170d0b757e91